packages/contracts-bedrock/src/dispute/SuperFaultDisputeGame.sol

```diff
@@ -187,13 +183,6 @@ contract SuperFaultDisputeGame is Clone, ISemver {
     /// @notice Flag for the `initialize` function to prevent re-initialization.
     bool internal initialized;
 
-    /// @notice Flag for whether or not the L2 block number claim has been invalidated via `challengeRootL2Block`.
-    bool public l2BlockNumberChallenged;
-
-    /// @notice The challenger of the L2 block number claim. Should always be `address(0)` if `l2BlockNumberChallenged`
-    ///         is `false`. Should be the address of the challenger if `l2BlockNumberChallenged` is `true`.
-    address public l2BlockNumberChallenger;
```

Comment:
(no action required)

For reviewers w/ less context on the setup here - yes, this changes the storage layout in a backwards-incompatible way, and this contract is proxied, but the proxies are never re-used. A fresh minimal proxy is created by the DGF every time we create a new dispute, so there's no risk of storage slot collisions in previous games.